var/spack/repos/builtin/packages/mvapich-plus/package.py

```diff
@@ -194,7 +197,7 @@ def file_system_options(self):
 
         fs = []
         for x in ("lustre", "gpfs", "nfs", "ufs"):
-            if "file_systems={0}".format(x) in spec:
+            if spec.satisfies("file_systems={x}"):
```

Comment:
```suggestion
            if spec.satisfies(f"file_systems={x}"):
```
I think this is also a simple typo in missing the fstring declaration.